test/SILGen/dependent_member_lowering.swift

```diff
@@ -9,13 +9,13 @@ struct Foo<T>: P {
   typealias A = T.Type
 
   func f(_ t: T.Type) {}
-  // CHECK-LABEL: sil hidden [transparent] [thunk] @_TTWurGV25dependent_member_lowering3Foox_S_1PS_FS1_1{{.*}} : $@convention(witness_method) <T> (@in @thick T.Type, @in_guaranteed Foo<T>) -> ()
-  // CHECK:       bb0(%0 : $*@thick T.Type, %1 : $*Foo<T>):
+  // CHECK-LABEL: sil hidden [transparent] [thunk] @_TTWurGV25dependent_member_lowering3Foox_S_1PS_FS1_1{{.*}} : $@convention(witness_method) <τ_0_0> (@in @thick τ_0_0.Type, @in_guaranteed Foo<τ_0_0>) -> ()
+  // CHECK:       bb0(%0 : $*@thick τ_0_0.Type, %1 : $*Foo<τ_0_0>):
```

Comment:
Well, we would have to at least drop the names on the type parameters coming from generic requirements (because they're moving to a new depth), and it felt weird to drop some but not all of them.